### ISSUE TITLE
finetuned_wgts_path error

### ISSUE THREAD

**Issue by ramakth1** [kaushaltrivedi/fast-bert#122]
Hi 
I have completed Language Model Fine-tuning and saved pytorch_model.bin to the new path

I am getting the below error 

The size of tensor a (12) must match the size of tensor b (512) at non-singleton dimension 4

 attention_scores = attention_scores+ attention_mask in modeling_bert.py

below are the sizes.
attention_scores torch.Size([1, 512, 12, 12, 12])
attention_mask torch.Size([12, 1, 1, 512])

how to address this issue ,, please help me .

Below is my code

from fast_bert.data_cls import BertDataBunch

databunch = BertDataBunch(DATA_PATH, LABEL_PATH,
                          tokenizer='bert-base-uncased',
                          train_file='train.csv',
                          val_file='val.csv',
                          label_file='labels.csv',
                          text_col='text',
                          label_col='label',
                          batch_size_per_gpu=16,
                          max_seq_length=512,
                          multi_gpu=True,
                          multi_label=False,
                          model_type='bert')

from fast_bert.learner_cls import BertLearner
from fast_bert.metrics import accuracy
import logging

logger = logging.getLogger()
device_cuda = torch.device("cuda")
metrics = [{'name': 'accuracy', 'function': accuracy}]

learner = BertLearner.from_pretrained_model(
						databunch,
						pretrained_path='bert-base-uncased',
						metrics=metrics,
						device=device_cuda,
						logger=logger,
						output_dir=OUTPUT_DIR,
						finetuned_wgts_path=new_path,
						warmup_steps=500,
						multi_gpu=True,
						is_fp16=True,
						multi_label=False,
						logging_steps=50)

learner.fit(epochs=6,
			lr=6e-5,
			validate=True, 	# Evaluate the model after each epoch
			schedule_type="warmup_cosine",
			optimizer_type="lamb")
